lib/loaders/gravity.js

```diff
@@ -22,6 +23,17 @@ load.with = (accessToken, loaderOptions = {}) => {
   };
 };
 
+load.authenticatedPost = (accessToken, loaderOptions = {}) => {
```

Comment:
This is explicitly for 'authenticated POST' requests to gravity, I added it rather than trying to be able to re-use any existing one.

No need for `DataLoader` or any of that stuff, its just a `Promise` wrapping a `request` (at the end of the day).